src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public enum Color {
+        Red,

```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
        RED,
```